modules/integrate-with-qcr-scanner.adoc

```diff
@@ -3,23 +3,27 @@
 // * integration/integrate-with-image-vulnerability-scanners.adoc
 :_module-type: PROCEDURE
 [id="integrate-with-qcr-scanner_{context}"]
-= Integrating with Quay Container Registry
+= Integrating with Quay Container Registry to scan images
 
 You can integrate {product-title} with Quay Container Registry for scanning images.
 
 .Prerequisites
-* You must have an OAuth token for authentication with the Quay Container Registry.
+* You must have an OAuth token for authentication with the Quay Container Registry to scan images.
 
 .Procedure
-. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
-. Under the *Image Integrations* section, select *Quay Container Registry*.
-+
-The *Configure image integration* modal box opens.
-. Click *New Integration*.
-. Fill in the required details for:
-.. *Integration Name*: The name of the integration.
-.. *Types*: Select *Scanner*.
-.. *Endpoint*: The address of the registry.
-.. *OAuth Token*: The OAuth token for Quay Container Registry.
-. Select *Test* (`checkmark` icon) to test that the integration with the selected registry is working.
-. Select *Create* (`save` icon) to create the configuration.
+
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Image Integrations* section, select *Red Hat Quay.io*.
+. Click *New integration*.
+. Enter the *Integration name.*
+. Under *Type*, select *Scanner* and enter information in the following fields:
```

Comment:
Is there a benefit to mentioning that you can do Registry + Scanner if you have both?